clientv3/README.md

```diff
@@ -10,7 +10,7 @@
 go get github.com/coreos/etcd/clientv3
 ```
 
-> Requires Go v1.9
+> **Requires Go v1.9+**
```

Comment:
remove `>` ?

Reply:
it makes this less significant.

Reply:
A quote is the closest thing to a "notice" in markdown. I think it draws more attention since it's a diff. color. But if you want it without no probs.

Reply:
@gyuho what do you think?

Reply:
do we need go 1.9 to build both etcd and etcd client? if that's the case, should we doc requirement on the front page?

Reply:
@fanminshi We mention Go 1.9 in front page for build.
@xiang90 I am fine with this change, since this markdown is only for people clicking github pages, not coreos.com.

Reply:
@gyuho I noticed that we mentioned in front page. I think mentioning go requirement here is redundant. If we add requirement here, we also need to make sure it is in sync which can be a pain.

Reply:
Then maybe just update https://github.com/coreos/etcd#getting-etcd?

> For those wanting to try the very latest version,

To

> For those wanting to try the very latest version/client,

?

Reply:
the doc https://github.com/coreos/etcd/blob/master/Documentation/dl_build.md#build-the-latest-version has outdated go version requirement.

Reply:
@fanminshi Yeah that needs to be fixed as well, in a separate patch.

Reply:
@gyuho @xiang90 I prefer updating the front page about build requirement instead doing here. Less place to track.

Reply:
@gyuho @fanminshi 

clientv3 is usually used as a standalone library. it worth adding a link here to the go version requirement if we want to avoid duplication.